state/migrations/firewallrules.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/juju/juju/core/firewall"
 )
 
+//go:generate mockgen -package migrations -destination firewallrules_mock_test.go -source=firewallrules.go
```

Comment:
These are located in `package_test.go`

Reply:
dammn, was looking at the test and not mock